src/lvl_script_lib.c

```diff
@@ -15,6 +15,7 @@
 #include "lvl_script_lib.h"
 #include "lvl_script_conditions.h"
 #include "lvl_script_commands.h"
+#include "lvl_script_value.h"
```

Comment:
since both new lines would be better in thing_list.h this include might not be needed then, so check if it does